0000-catalog.md

```diff
@@ -0,0 +1,789 @@
+- Feature Name: Product Catalog
+- Start Date: 2019-07-12
+- RFC PR:
+- Hyperledger Grid Issue: [GRID-95](https://jira.hyperledger.org/browse/GRID-95)
+
+# Summary
+
+[summary]: #summary
+
+This RFC proposes a generic implementation for a Hyperledger Grid _Product
+Catalog_. Product Catalogs will be represented by a unique identifier, which
+will be referenced by the _catalog_products_ included in the catalog. That
+catalog_id can be used to share a grouping of products that can be shared with 
+one or more organizations. The Grid Catalog will contain 4 fields. A 
+`catalog_id`, a `catalog_owner`, a `name` for the catalog, and a repeated 
+field of PropertyValues for custom fields.
+
+In addition, a base catalog product schema is included as a way to demonstrate
+enforcement of additional product properties. Any grid product added to a
+catalog will adhere to the "catalog_product" schema (or another custom-defined 
+schema). The example schema in particular will enforce a required `catalog_id`
+field, a required `status` field, an optional `return_policy`, and an 
+optional `price` field.
+
+# Motivation
+
+[motivation]: #motivation
+
+The Grid Catalog implementation is designed for sharing an assortment of
+products between organizations. Each product in a catalog will contain a number
+of master data elements. Using the Grid Catalog, organizations can share
+collections of related/unrelated products with other participants in their
+network.
+
+# Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+## Entities
+
+A `catalog` is a high-level construct that `catalog_products` can
+reference. A _catalog_product_ is an instance of a Grid Product with additional 
+schema defined attributes specific to the trading partner involved. The 
+"catalog_product" schema can be extended to include any number of attributes that 
+are required by an organization. A catalog can be referenced, or used to share 
+item level information in a supply chain. A catalog is referenced by a 
+"catalog_product" via its `catalog_id`. A catalog also has an `owner` (the 
```

Comment:
Above, `owner` is called `catalog_owner`. If these are pointing to the same parameter, the name should be consistent.

Reply:
✅